.github/workflows/pull-request-lint.yml

```diff
@@ -36,6 +36,7 @@ jobs:
             docs
             chore
             rfc
+            revert
```

Comment:
Let's also update the contributor's guide https://docs.winglang.io/contributors/pull_requests#how-are-pull-request-titles-formatted

Reply:
#1728